web/src/components/core/code/MqCode.tsx

```diff
@@ -8,13 +8,11 @@ import withStyles, { WithStyles } from '@material-ui/core/styles/withStyles'
 
 const styles = (theme: Theme) =>
   createStyles({
-    // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
-    // @ts-ignore
     codeContainer: {
       padding: `${theme.spacing(2)}px ${theme.spacing(4)}px`,
       backgroundColor: alpha(theme.palette.common.white, 0.1),
       borderLeft: `2px dashed ${THEME_EXTRA.typography.subdued}`,
-      whiteSpace: 'break-spaces'
+      whiteSpace: 'pre-wrap'
```

Comment:
One of the upgraded TypeScript linters didn't like this CSS property, so we've made a corresponding change. 

The documentation is available here. https://developer.mozilla.org/en-US/docs/Web/CSS/white-space